nilearn/plotting/surf_plotting.py

```diff
@@ -226,6 +231,8 @@ def _plot_surf_plotly(coords, faces, surf_map=None, bg_map=None,
             bg_data, colors["cmap"], colors["norm"],
             colors["abs_threshold"]
         )
+
+    # add colorbar
```

Comment:
I wonder if some of these comments are not redundant with the code. also this one is misplaced -- the 2 lines below it create the main figure content (the Mesh3d) and are not related to the colorbar

Reply:
I got rid of this one since what's going on below is obvious, but I generally think short single-line comments don't hurt :slightly_smiling_face: